website/static/js/comment.js

```diff
@@ -291,6 +511,50 @@ var CommentModel = function(data, $parent, $root) {
         self.author = self.$root.author;
     }
 
+    self.editableContent = ko.computed(function() {
+        var content = self.content();
+        if (!content) {
+            content = '';
+        }
+        var regex = /\[(@|\+)(.*?)\]\(\/([a-z\d]{5})\/\)/;
+        var matches = content.match(/\[(@|\+)(.*?)\]\(\/([a-z\d]{5})\/\)/g);
+        if (matches) {
```

Comment:
Why do we need this `if` here? Won't the loop on the next line simply run zero times if `matches.length === 0`? Or is `matches` something besides an Array in the case of a match failure?